dotnetv3/Aurora/README.md

```diff
@@ -1,88 +1,131 @@
+<!--Generated by WRITEME on 2023-02-16 16:08:14.360750 (UTC)-->
 # Aurora code examples for the SDK for .NET
 
 ## Overview
-The code examples in this section show how to use the AWS SDK for .NET with Amazon Aurora
-to create and manage Amazon Aurora DB clusters, DB instances, and custom parameter groups.
 
-Amazon Relational Database Service (Amazon RDS) is a web service that makes it easier
-to set up, operate, and scale a relational database in the cloud. Aurora is a fully
-managed relational database engine that's compatible with MySQL and PostgreSQL.
+Shows how to use the AWS SDK for .NET to work with Amazon Aurora.
+
+<!--custom.overview.start-->
+<!--custom.overview.end-->
+
+*Aurora is a fully managed relational database engine that's built for the cloud and compatible with MySQL and PostgreSQL. Amazon Aurora is part of Amazon Relational Database Service (Amazon RDS).*
+
+## ⚠ Important
 
-## ⚠️ Important
 * Running this code might result in charges to your AWS account.
 * Running the tests might result in charges to your AWS account.
 * We recommend that you grant your code least privilege. At most, grant only the minimum permissions required to perform the task. For more information, see [Grant least privilege](https://docs.aws.amazon.com/IAM/latest/UserGuide/best-practices.html#grant-least-privilege).
 * This code is not tested in every AWS Region. For more information, see [AWS Regional Services](https://aws.amazon.com/about-aws/global-infrastructure/regional-product-services).
 
+<!--custom.important.start-->
+<!--custom.important.end-->
+
 ## Code examples
 
 ### Get started
 
-* [Hello Aurora](Actions/HelloAurora.cs)
+* [Hello Aurora](Actions/HelloAurora.cs#L4) (`DescribeDBClusters`)
```

Comment:
This looks as though something was tacked on accidentally.

Reply:
No, this is the line linking that is done by the Jinja generation. It provides the line-level linking that is one of the benefits of using the WRITEME generated files.